streams/src/main/java/org/apache/kafka/streams/tools/KafkaStreamsCleanupClient.java

```diff
@@ -0,0 +1,249 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements. See the NOTICE
+ * file distributed with this work for additional information regarding copyright ownership. The ASF licenses this file
+ * to You under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.apache.kafka.streams.tools;
+
+import joptsimple.OptionException;
+import joptsimple.OptionParser;
+import joptsimple.OptionSet;
+import joptsimple.OptionSpec;
+import kafka.admin.AdminUtils;
+import kafka.admin.TopicCommand;
+import kafka.utils.ZkUtils;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.security.JaasUtils;
+import org.apache.kafka.common.serialization.ByteArrayDeserializer;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * TODO
+ */
+public class KafkaStreamsCleanupClient {
+    private static OptionSpec<String> bootstrap_server_option;
+    private static OptionSpec<String> zookeeper_option;
+    private static OptionSpec<String> application_id_option;
+    private static OptionSpec<String> source_topics_option;
+    private static OptionSpec<String> intermediate_topics_option;
+    private static OptionSpec<String> operator_option;
+    private static OptionSpec<String> window_option;
+    private static OptionSpec<String> state_dir_option;
```

Comment:
You might want to change the names of these to be standard Java naming, i.e, camel case. I'd be surprised if Checksytle doesn't complain about these.